tests/seed_isort_config_test.py

```diff
@@ -95,12 +99,72 @@ def test_integration_extra_file(tmpdir):
         assert tmpdir.join('.isort.cfg').read() == expected
 
 
-def test_integration_no_config(tmpdir, capsys):
+@pytest.mark.parametrize(
+    'initial_filesystem,expected_filesystem',
```

Comment:
please use the tuple format for parametrizing, stringly-typed--

Reply:
Sure, was copying most of the examples in https://docs.pytest.org/en/latest/example/parametrize.html 🙈